src/vscode-dts/vscode.proposed.contribIssueReporter.d.ts

```diff
@@ -0,0 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// empty placeholder declaration for the `issue reporter`-submenu contribution point
+// https://github.com/microsoft/vscode/issues/196863
```

Comment:
Do we usually do empty proposed files for menu contribution changes? I didn't think menu contribution had the same scrutiny

Reply:
yeah was following what Joh sampled for menu contribution points. as I mentioned, it works without, so not sure if we need or if there's something going wrong
 
https://github.com/microsoft/vscode/blob/e70ab0f56a4d8177c67deb5e3606e64d1b2f4422/src/vscode-dts/vscode.proposed.contribShareMenu.d.ts#L0-L1

Reply:
These empty files look funny but that's the only way to enforce their proposals because the underlying check-function is strictly typed to the names of proposals